akka-stream-tests/src/test/scala/akka/stream/impl/fusing/LifecycleInterpreterSpec.scala

```diff
@@ -138,4 +140,54 @@ class LifecycleInterpreterSpec extends InterpreterSpecKit {
 
   }
 
+  private[akka] case class PreStartAndPostStopIdentity[T](
```

Comment:
This is just a move from InterpreterSpecKit, but I have not yet removed it from there.